mypy/options.py

```diff
@@ -113,6 +117,9 @@ def __init__(self) -> None:
         self.debug_cache = False
         self.quick_and_dirty = False
 
+        # plugins
+        self.plugins = []  # type: List[PluginRegistry]
```

Comment:
My guess is that this is because of a reference cycle. There is a known bug that mypy does not treat named tuples in cycles correctly.